react/scss/styles.scss

```diff
@@ -0,0 +1,114 @@
+.bookSearch {
+  max-width: 792px;
+  margin: 0 auto;
+}
+
+.bookItem {
```

Comment:
I'd split this file into many `.scss` / `.css`, one per component. So in each folder component we could have:
```
index.js
styles.scss or styles.css
```
In each component, we will need to:
```
import './styles.css';  // <-- Add this without assigning it a variable

class MyComponent extends Component {
 ...
}
```